minecraft_servers/traidio/manifest.json

```diff
@@ -0,0 +1,102 @@
+{
+  "server_name": "traidio",
+  "nice_name": "TraidioNET",
+  "direct_ip": "traidio.net",
+  "server_wildcards": [
+    "%.traidio.net"
+  ],
+  "supported_languages": [
+    "de",
+    "en"
+  ],
+  "social": {
+    "web": "https://panel.traidio.net/overview",
```

Comment:
If website is locked behind Discord authentication, remove

Reply:
I think you meant `https://traidio.net` as your website?